api/pom.xml

```diff
@@ -287,26 +287,18 @@
 	   <dependency>
 		   <groupId>org.junit.jupiter</groupId>
 		   <artifactId>junit-jupiter-api</artifactId>
-		   <version>${junitVersion}</version>
```

Comment:
can you please check if you now do a `mvn dependency tree` are these libs still with scope test?